sphinx/ext/intersphinx.py

```diff
@@ -287,7 +287,7 @@ def load_mappings(app):
 
 
 def missing_reference(app, env, node, contnode):
-    # type: (Sphinx, BuildEnvironment, nodes.Node, nodes.Node) -> None
+    # type: (Sphinx, BuildEnvironment, nodes.Element, nodes.TextElement) -> None
```

Comment:
I guess `node` might be `nodes.reference` class. I'll take a look in another PR.